_posts/2021-01-29-design-system-schizophrenia.md

```diff
@@ -0,0 +1,110 @@
+means switching from the traditional "designers vs developers" mindset (left) to
+a Product mindset (right). We don't like switching mindset.
+
+To install a  Design System what you want to target is having a *team* that
+owns a *product* (the DS and its artifacts) that have *customers* (your others
+teams).
+
+But, I'm forced to admit that I don't see a lot of Design System *teams* in
+companies.
+
+There are Design System *projects*, for sure. It's very trendy those days. They
+are mainly driven by the base of the organization, ie. some designers and/or
+developers. They work on it on the week-end. They come early in the morning
+or they leave late at night to have some time to code for the Design System
+besides their "real" job, shipping features.
+
+Sometimes, when those Design System projects begin to see some traction
+(because, yeah, it's a powerful concept) and when the initial volunteers haven't
+burned out yet, leadership enter the game and declare "We have a Design
+System! It's so good! Thanks so and so.".
+They write a blog post showing the world how their Design System revolutionized
+their company while in fact nothing happen inside.
+
+Nothing happen inside because they haven't embraced that a Design System is a
+product that need a dedicated team.
+
+It's far from impossible to form such a team.
+We already have infrastructure/platform teams that give others a solid platform
+to run their services. We have security teams that give others tools to build
+secure services. We have engineering productivity teams that help others with
+day to day tools (CI, testing environment, etc.). We have analysts and UX
+researchers to support others at making decisions based on data.
+
+We have all those teams but you can't form one for the Design System? What's
+your problem?
+
+Is it that you have to group designers and developers together and that's new
+for you?
+Don't you already do that for feature teams, squads, swarms or whatever name you
+give to your multidiciplinary teams? Those that work on features.
+
+Is it that you don't want to start a war by having to decide whether this team
+reports to the VP Product or the VP Engineering? Does it really matter?
+
+Is it because your designers or developers are already understaffed? Don't you
+see how such a organization could be exactly what you need to accomodate with
+that?
+
+Is it because you don't have a PO/PM to lead this team? Don't you think you can
+just start with those ones that have managed to build the current Design System
+while doing their day to day job *at the same time*?
+
+By wanting your company to produce a Design System but not updating your
+organization to support it, you're making your designers and developers go
+crazy.
+
+They have to fight every day with their management for this fondamental product
+because it's *never* the priority and there is *never* enough room on the
+roadmap for it.
```

Comment:
```suggestion
They have to fight every day with their management for this fondamental product
because it's *never* the priority and there is *never* enough room on the
roadmap for it. But they see its benefits, so they keep trying, and trying, and trying.
```